src/components/Tab.ts

```diff
@@ -0,0 +1,61 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { Clickable } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+
+const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    marginTop: '1rem',
+    height: '3.75rem',
+  },
+  button: (isActive: boolean) => ({
+    ...Style.tabBar.tab,
+    ...(isActive ? Style.tabBar.active : {}),
+    backgroundColor: undefined,
+    fontSize: 14,
+  }),
+};
+
+type TabProps = {
+  key: string;
+  title: string;
+  isValid: boolean;
+  activeTabKey: string;
+  onChangeTab: (arg0: string) => void;
+};
+
+const Tab = ({ key, title, isValid, activeTabKey, onChangeTab }: TabProps) =>
+  h(
```

Comment:
Functions should also specify the return type.

```suggestion
const Tab = ({ key, title, isValid, activeTabKey, onChangeTab }: TabProps): ReactNode =>
  h(
```

Also (and this is nit picky, so ignore it if you like), I would suggest a convention like the following:

```suggestion
const Tab = (props: TabProps): ReactNode => {
  const { key, title, isValid, activeTabKey, onChangeTab } = props;
  return h(
```

It's easier to see the type of the arguments. With props destructured in line, it gets kind of lost in the clutter.

I find it's more future proof. If, later, the number of props increases to where they no longer fit on one line, or we need to add a hook to this component, this style needs a smaller diff (which is easier to understand and review) to accommodate those changes.